src/engine/renderer/tr_local.h

```diff
@@ -1324,9 +1324,18 @@ static inline void glFboSetExt()
 	  SHADER_LIGHT // light material: attenuation
 	};
 
+	// Size of MAX_QPATH + "$depth".
+	const int depthShaderNameLen = MAX_QPATH + 6;
+
 	struct shader_t
 	{
-		char         name[ MAX_QPATH ]; // game path, including extension
+		/* Shader path, including optional internal depth suffix and extension.
+		The code should always check for shader name to not be longer than MAX_QPATH
+		except when checking for depthShader names. This string is longer than
+		MAX_QPATH for the only reason we add a suffix to existing shader names when
+		creating a depth shader. */
+		char name[ depthShaderNameLen ];
```

Comment:
Will this allow any shader to be longer than MAX_QPATH? will this cause any issues if we have other buffers that limit to MAX_QPATH?

Reply:
I reverted to the simpler check where this string is always MAX_QPATH long.